spec/support/shared_examples/create_service_with_price_rule_shared_examples.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.shared_examples "creates a product with billing mode" do |product_type, billing_mode|
+  let(:logged_in_user) { administrator }
+
+  it "can create a #{product_type}, which automatically creates a price rule" do
+    visit facility_products_path(facility)
+    all("a", text: product_type.capitalize)[0].click
+    click_link "Add #{product_type.capitalize}"
+
+    fill_in "Name", with: "My new Product", match: :first
+    fill_in "URL Name", with: "new-service"
+    select "Required", from: "#{product_type}[user_notes_field_mode]"
+    select billing_mode, from: "#{product_type}[billing_mode]"
```

Comment:
Can we do `choose billing_mode`?

Reply:
I don't think so, as this is a drop-down, not a radio button

Reply:
You're right 😅 

What about something like `select billing_mode, from: "Billing Mode"`?